api/CanvasCaptureMediaStream.json

```diff
@@ -0,0 +1,162 @@
+{
+  "api": {
+    "CanvasCaptureMediaStream": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CanvasCaptureMediaStream",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": "51"
+          },
+          "chrome_android": {
+            "version_added": false
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "41",
+            "notes": "Disabled by default; set the preference <code>canvas.capturestream.enabled</code> to <code>true</code> to activate."
```

Comment:
We don't use note for indicating disabled by default feature. See https://github.com/mdn/browser-compat-data/blob/master/compat-data-schema.md#flag for using the "flag" object.